http-plugins/src/test/java/co/cask/hydrator/plugin/batch/HttpCallbackActionTest.java

```diff
@@ -96,9 +96,9 @@ public static void setupTestClass() throws Exception {
     List<HttpHandler> handlers = new ArrayList<>();
     handlers.add(new MockFeedHandler());
     httpService = NettyHttpService.builder("MockService")
```

Comment:
Since `NettyHttpService` is used directly, you should have it declared as direct dependecy

Reply:
I mean a direct dependency on the cask netty-http library

Reply:
ok; done